<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSA</class>
 <widget class="QDialog" name="RSA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="outputEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>521</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Encrypt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputPLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>30</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputQLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DeBug</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setPQ">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>generatePQ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="InputEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>521</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Decrypt">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
